tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -1044,8 +1042,8 @@ class MklConvOp : public OpKernel {
       AllocateOutputTensor(context, *conv_fwd_pd, dst_dims_mkl_order, tf_fmt,
                            &dst_tensor);
       Tensor* filter_out_tensor = nullptr;
-      if (typeid(Tinput) == typeid(float) && typeid(Tfilter) == typeid(float) &&
-          typeid(Toutput) == typeid(float)) {
+      if (typeid(Tinput) == typeid(float)&&typeid(Tfilter) ==
+          typeid(float)&&typeid(Toutput) == typeid(float)) {
```

Comment:
This formatting looks weird. Did you use a different formatter than `clang-format`? Could you please try applying `clang-format`?

Reply:
@penpornk Do you know if there is a specific version of clang-format that we need to use? I use 3.4.2 version with the command: 
clang-format <file_name> --style=google

Reply:
What about without `--style=google`?

Reply:
@penpornk Thanks! It looks like I was running an old version of clang-format. I re-ran a newer version. Please have a look. The actual bug fix is at line: https://github.com/tensorflow/tensorflow/pull/24021/files#diff-192dfeafbdd684934bdb0dfa8983674aR1064
I will run the newer version for #23724 as well.

Reply:
Thank you very much! This looks great! (Sorry I couldn't tell you the version earlier. I don't know what version ours is based on.)